### PR TITLE
Feat: add safe logo to header bar on mobile

### LINKED ISSUE #3932: Adjust header for mobile viewports
## What is the feature about

There is no Safe logo visible on mobile viewports and the existing header elements take up more space than necessary

## The list of requirements

- Add small Safe logo on mobile viewports
- Hide the network name in the network selector on mobile viewports

## Designs/sketches
Screenshot 2024-07-10 at 17 46 02

## Links

- [Figma](https://www.figma.com/design/HKW5iarMST9rL7Kxkf59qY/MUI-for-Safe?node-id=9112-89398&t=kep64IFqizgo5WCn-0)

### PR DESCRIPTION
**Pull request by jmealy**
## What it solves

Resolves #3932

## How this PR fixes it
- Adds small Safe logo on mobile viewports
- Removes the chain name for the network selector on mobile viewports

## How to test it
- Open a safe on a mobile screen size
- Check that the header contains the new logo which behaves the same as the larger logo.
- Check that the header is displayed correctly for all screen sizes.

## Screenshots
Disconnected: 
image

Connected:
image

## Checklist
* [ ] I've tested the branch on mobile 📱
* [ ] I've documented how it affects the analytics (if at all) 📊
* [ ] I've written a unit/e2e test for it (if applicable) 🧑‍💻

### CODE REVIEW

**Comment by katspaugh** [comment]
Does it fit with a connected wallet?

**Comment by usame-algan** [comment]
One minor thing: Can we increase the margin-left in case there is only the logo and no menu icon to 16px instead of 8px? Looks a bit too far to the left here
Screenshot 2024-07-19 at 17 21 15

File: src/components/common/NetworkSelector/styles.module.css [hunk 1]
```diff
@@ -28,9 +28,9 @@
   pointer-events: none;
 }
 
-.menuItem {
-  padding-top: var(--space-1);
-  padding-bottom: var(--space-1);
+.select :global .MuiMenuItem-root {
+  padding: 0;
+  pointer-events: none;
```
Comment (usame-algan) [inline src/components/common/NetworkSelector/styles.module.css:33]: Why do we need to set pointer-events: none?

**Review by usame-algan — APPROVED** [review]
Thanks!

File: src/components/common/NetworkSelector/styles.module.css (hunk 1 above)
Comment (jmealy) [reply src/components/common/NetworkSelector/styles.module.css:33]: when you add this `renderValue` prop to Select it wraps each item in an li and adds a hover highlight

**Comment by francovenica** [comment]
I'm not seeing the logo at all
I tried the /welcome page, /welcome/accounts, inside of a safe...
I tried emulating with the web browers "mobile viewport" and with an iPhone I have

image

**Comment by usame-algan** [comment]
Probably it wasn't deployed due to the issue we had with our CI. Should be fixed once dev is pulled into this Branch again.

**Comment by jmealy** [comment]
> Probably it wasn't deployed due to the issue we had with our CI. Should be fixed once dev is pulled into this Branch again.

@francovenica can you check again?

**Comment by francovenica** [comment]
Yes, I can see it now. LGTM

image
image

**Pull request merged**